packages/common/src/components/utils/InfoBox.tsx

```diff
@@ -54,10 +59,10 @@ const InfoBox: React.FC<Props> = ({ type, children, additionalStyles, squared, d
 
   return (
     // @ts-expect-error squared is giving error but it's working as intended
-    <StyledWrapper squared={squared} className={`ic-alert ic-alert--status ${getWrapperClass()} ${additionalStyles}`}>
-      <Wrapper className={`${deactivateIconWrap ? 'iu-flex' : ''} ${additionalWrapperStyles}`}>
-        <Icon className={`${deactivateIconWrap ? '' : 'iu-fl'} ic-alert__icon ${getIconClass()}`}></Icon>
-        <div className={`${deactivateIconWrap ? '' : 'iu-ml-300'}`}>{children}</div>
+    <StyledWrapper squared={squared} className={`ic-alert ic-alert--status iu-lh-body ${getWrapperClass()} ${additionalStyles}`}>
+      <Wrapper className={`${!activateIconWrap ? 'iu-flex' : ''} ${additionalWrapperStyles}`}>
+        <Icon activateIconWrap={!!activateIconWrap} className={`${!activateIconWrap ? '' : 'iu-fl'} ic-alert__icon ${getIconClass()}`} />
+        {activateIconWrap ? <div className={'iu-ml-300'}>{children}</div> : <>{children}</>}
```

Comment:
Jag tycker att det är lite svårläst när man villkorar klassnamn i template literals på det här sättet. Om detta är något vi gör ofta föreslår jag att vi drar in paketet `classnames` (https://github.com/JedWatson/classnames#readme) vilket underlättar denna typ av hantering och gör det lite mer lättläst.